runtime/src/lib.rs

```diff
@@ -519,8 +519,15 @@ impl additional_traits::DelegatedDispatchVerifier<CennznetDoughnut> for Runtime
 			.ok_or("CENNZnut does not grant permission for cennznet domain")?;
 		let cennznut: CENNZnut = Decode::decode(&mut domain).map_err(|_| "Bad CENNZnut encoding")?;
 
-		// Strips [c|p|s]rml- prefix
-		match cennznut.validate(&module[5..], method, &[]) {
+		// Extract Module name from <prefix>-<Module_name>
+		let dash_pos;
+		match module.find('-') {
```

Comment:
Can make it more idiomatic like this:
```
let module_offset = module.find('_').ok_or("error during module name segmentation")? + 1;
```

Reply:
Can be `let _ = module.find...` to avoid unused variable warning as well

Reply:
shortened the code as Jordy suggested, it get rids of the warning at the same time